projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -89,7 +108,7 @@ private Project createProject(String code, String title,
     ProjectCode projectCode;
     try {
       projectCode = ProjectCode.parse(code);
```

Comment:
Why cant we fail earlier? Where does the project code come from and should we not fail already earlier, if the format is violated?

Reply:
The project code format is checked int he `ProjectCode#parse` method